sections/main-search.liquid

```diff
@@ -20,7 +20,7 @@
 
   .template-search__search {
```

Comment:
Group headings are centered on the search page.
<img src="https://screenshot.click/10-42-2lu7g-mklob.png" width="300" />